### PR TITLE
Support for SRP BlobServiceProperties API

### PR DESCRIPTION
**Pull request by huizlAzure**
This checklist is used to make sure that common issues in a pull request are addressed. This will expedite the process of getting your pull request merged and avoid extra work on your part to fix issues discovered during the review process.

### PR information
- [ ] The title of the PR is clear and informative.
- [ ] There are a small number of commits, each of which have an informative message. This means that previously merged commits do not appear in the history of the PR. For information on cleaning up the commits in your pull request, [see this page](https://github.com/Azure/azure-powershell/blob/master/documentation/development-docs/cleaning-up-commits.md).
- [ ] Except for special cases involving multiple contributors, the PR is started from a fork of the main repository, not a branch.
- [ ] If applicable, the PR references the bug/issue that it fixes.
- [ ] Swagger files are correctly named (e.g. the `api-version` in the path should match the `api-version` in the spec).

### Quality of Swagger
- [ ] I have read the [contribution guidelines](https://github.com/Azure/azure-rest-api-specs/blob/master/.github/CONTRIBUTING.md).
- [ ] My spec meets the review criteria:
  - [ ] The spec conforms to the [Swagger 2.0 specification](https://github.com/OAI/OpenAPI-Specification/blob/master/versions/2.0.md).
  - [ ] The spec follows the guidelines described in the [Swagger checklist](https://github.com/Azure/azure-rest-api-specs/blob/master/documentation/swagger-checklist.md) document.
  - [ ] [Validation tools](https://github.com/Azure/azure-rest-api-specs/blob/master/documentation/swagger-checklist.md#validation-tools-for-swagger-checklist) were run on swagger spec(s) and have all been fixed in this PR.

### CODE REVIEW

**Comment by hovsepm** [comment]
@ravbhatnagar could you review this PR? It adds new endpoint to an existing stable Storage Blob APIs.

**Comment by blueww** [comment]
@hovsepm ， @ravbhatnagar 
I don't see the PR add any new endpoint. It add 2 new APIs.

**Comment by ravbhatnagar** [comment]
@vladimirjoanovic for review from ARM side.

File: specification/storage/resource-manager/Microsoft.Storage/stable/2018-07-01/blob.json [hunk 1]
```diff
@@ -16,6 +16,91 @@
       "application/json"
     ],
     "paths": {
+      "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.Storage/storageAccounts/{accountName}/blobServices/{BlobServicesName}": {
+       "put": {
+        "tags": [
+         "BlobService"
+        ],
+        "operationId": "BlobService_SetServiceProperties",
```
Comment (hovsepm) [inline specification/storage/resource-manager/Microsoft.Storage/stable/2018-07-01/blob.json]: >BlobService [](start = 24, length = 11)

Please make the word plural. Should be "BlobServices_"

File: specification/storage/resource-manager/Microsoft.Storage/stable/2018-07-01/blob.json [hunk 2]
```diff
@@ -917,6 +1002,108 @@
...
+      "DeleteRetentionPolicy": {
+        "properties": {
+          "enabled": {
+            "type": "boolean",
+            "description": "Indicates whether DeleteRetentionPolicy is enabled for the Blob service."
+          },
+          "days": {
+            "type": "integer",
+            "minimum": 1,
+            "maximum": 365,
```
Comment (hovsepm) [inline specification/storage/resource-manager/Microsoft.Storage/stable/2018-07-01/blob.json]: >365 [](start = 23, length = 3)

just a note - in a leap year there are 366 days. Does it matter in your case?

**Comment by hovsepm** [comment]
Please fix java section in readme. The multitag pointer there does not include this api version and that is why no java code is generated during CI.

File: specification/storage/resource-manager/Microsoft.Storage/stable/2018-07-01/blob.json (hunk 2 above)
Comment (huizlAzure) [reply specification/storage/resource-manager/Microsoft.Storage/stable/2018-07-01/blob.json]: 365 is intended max value as specified by underlying data-plane API. Thanks for noting!

**Review by hovsepm — APPROVED** [review]

**Comment by jhendrixMSFT** [comment]
@AutorestCI regenerate azure-sdk-for-go

**Pull request merged**